src/components/Global/ButtonIcon.vue

```diff
@@ -15,6 +21,9 @@ export default defineComponent({
     close() {
```

Comment:
I would suggest naming this `closed` instead of `close` - close suggests an action, so a method.
Likewise with the `open` 😄